doc/manual/index.rst

```diff
@@ -59,4 +59,5 @@ source code and issue tracker
     browser-gui
     network
     ssr-as-library
+    use-cases
```

Comment:
Since the use cases only use the stand-alone SSR (AFAICT), this section could come before `ssr-as-library`.

Actually, now that I think about it, wouldn't most of this stuff be much easier using the SSR Pd externals?

Admittedly, the BRS renderer wasn't available until a few days ago (#266), but isn't this a game changer?

No multiple SSR instances needed, just a few `[ssr_brs~]` objects. No connecting and disconnecting. No start script needed.

And Windows support! Well, that's quite untested ...

I guess the main difference is that the stand-alone SSR supports head tracking, but apart from that everything else should be better with the Pd externals, right?

AFAICT, the examples don't use the head tracker support of the stand-alone SSR, or do they?

Reply:
I think that the order is fine. First come all the technical things, then come the `use-cases`. 

The BRS renderer in Pd is indeed cool. I'd say that we should add it (but keep also the current use cases with the stand-alone SSR). I'll take care of that.

No, no head tracking on the multiple-instance SSR. This works only with the Pd patch [tracker_to_4_ssr.pd](https://github.com/SoundScapeRenderer/listening-test/blob/efc11f8642f6174b93c83c0c3610ea4263224671/data_linked_in_ssr_manual/tracker_to_4_ssr.pd) because as soon as an SSR instance reads tracker data, the access to that stream is blocked.